.github/workflows/pr-checks.yml

```
@@ -8,59 +8,58 @@ jobs:
-      run: |
-        # Get the PR title and check if it follows conventional commits
-        PR_TITLE="${{ github.event.pull_request.title }}"
-        if ! echo "$PR_TITLE" | grep -qE "^(feat|fix|docs|style|refactor|perf|test|chore|ci)(\(.+\))?: .+"; then
-          echo "::warning::PR title should follow conventional commit format: type(scope): description"
-        fi
+      - name: Verify build works
+        run: npm run build
 
+      - name: Check commit message format
+        if: github.event_name == 'pull_request'
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)